### DISCUSSION TITLE
Non-zero status code returned while running Conv node.

### DISCUSSION THREAD

**Discussion by ZZHHogan** [microsoft/onnxruntime#6170] [Other Q&A]
When I running my project,it can run normally.But after I generate DLL files from this project ,the DLL can not run in win10,x64 operation system.Whereas,it can run in x86 operation sysytem when generated from x86 operation system setting.And the code and the mode I use is same.
The bug shown is :
![image](https://user-images.githubusercontent.com/40531744/102562346-d80e5700-4111-11eb-9370-074f2784901d.png)

**Comment by snnn** [comment]
Similar as #6114

It must because you didn't create the Ort::Env or you deleted it .

**Reply by ZZHHogan** [reply]
> Similar as #6114
> 
> It must because you didn't create the Ort::Env or you deleted it .

OK，Thanks，I will check my code later.
